index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <title></title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <!--script src='main.js'></script-->
+</head>
+<body>
+    <nav class="toolbar">
+        <img id="img1" src="image_geometry_1.svg" alt="">
+        <div class="ic_icon">
+          <button id="btn"></button>
+          <button id="btn"></button>
+          <button id="btn"></button>
+        </div>
```

Comment:
Your design (Navbar)
![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/78251927/e86eaf9b-cd74-412c-bcce-c068071bbc89)

Figma
![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/78251927/0bab49e3-d1b9-4ad9-ba56-a47b500fe5b0)

As requested by the previous reviewer, please adjust your design so that it looks exactly like the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), 

Part of our learning objective is to understand how to parse a Figma design to create a UI. 💪 🤗